actions.js

```diff
@@ -253,10 +261,51 @@ const ThingsAction = Java.type('org.openhab.core.model.script.actions.Things');
  */
 const Voice = Java.type('org.openhab.core.model.script.actions.Voice');
 
-[Audio, BusEvent, Ephemeris, Exec, HTTP, LogAction, Ping, ScriptExecution, Semantics, ThingsAction, Voice].forEach(function (item) {
-  exports[item.class.getSimpleName()] = item.class.static;
-});
+/**
+ * Cloud Notification Actions
+ *
+ * If the {@link https://www.openhab.org/addons/integrations/openhabcloud/ openHAB Cloud Connector} add-on is installed, notifications can be sent to registered users/devices.
+ *
+ * The static methods of this class are made available as functions in the scripts. This allows a script to send notifications using the openHAB Cloud Connector add-on.
+ * See {@link https://www.openhab.org/docs/configuration/actions.html#cloud-notification-actions Cloud Notification Action Docs} for full documentation.
+ *
+ * @example
+ * NotificationAction.sendNotification('<email>', '<message>'); // to a single myopenHAB user identified by e-mail
+ * NotificationAction.sendBroadcastNotification('<message>'); // to all myopenHAB users
+ * NotificationAction.sendLogNotification('<message>'); // only listed in the notification log
+ *
+ * @name NotificationAction
+ * @memberof actions
+ */
+let NotificationAction;
+try {
+  NotificationAction = Java.type('org.openhab.io.openhabcloud.NotificationAction');
+} catch (error) {
+  if (error.name !== 'TypeError') throw Error(error);
```

Comment:
I'm assuming if the cloud binding is not installed it will throw this 'TypeError' ?

Reply:
Yes.
When the binding is not installed, the loading of the class fails with the TypeError.

Reply:
This error is caught, other errors are rethrown.